lxd/network/acl/driver_common.go

```diff
@@ -751,33 +751,48 @@ func (d *common) Delete() error {
 // GetLog gets the ACL log.
 func (d *common) GetLog(clientType request.ClientType) (string, error) {
 	// ACLs aren't specific to a particular network type but the log only works with OVN.
-	logPath := shared.HostPath("/var/log/ovn/ovn-controller.log")
-	if !shared.PathExists(logPath) {
-		return "", fmt.Errorf("Only OVN log entries may be retrieved at this time")
-	}
+	var logEntries []string
+	var err error
+	prefix := fmt.Sprintf("lxd_acl%d-", d.id)
```

Comment:
can the scope of the prefix var be reduced by moving it into one of the if statements?